src/data/communities.json

```diff
@@ -175,5 +175,16 @@
     "linkedin": "https://www.linkedin.com/company/codeonjvmchennai/",
     "telegram": "https://t.me/codeonjvmchennai",
     "youtube": "https://www.youtube.com/channel/UCxQft97E-LVwIHcZd38JghA"
+  },
+  {
```

Comment:
Hey @hackeronedilip5511 
we merged the event update, it would be great if you could pull a PR with just the community file changes

Cheers :)

Reply:
Hey @hackeronedilip5511 any updates buddy ?